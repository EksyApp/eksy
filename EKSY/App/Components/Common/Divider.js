import React, {Component} from 'react'
import * as Elements from 'react-native-elements'
import {StyleSheet} from 'react-native'
import * as Theme from '../../Theme'

export class Divider extends Component {

	render() {
		return(
				<Elements.Divider {...this.props} style={[style.divider, this.props.style]}/>
		)
	}

}

const style = StyleSheet.create({
	divider: {
		height: 1,
		margin: 3,
		marginLeft: 10,
		marginRight: 10,
		backgroundColor: Theme.detailColor
	}
})

export default Divider
