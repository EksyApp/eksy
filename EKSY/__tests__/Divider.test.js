import 'react-native';
import React from 'react';
import Divider from '../App/Components/Divider';
import renderer from 'react-test-renderer';

test('divider renders correctly', () => {
  const tree = renderer.create(
    <Divider />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
