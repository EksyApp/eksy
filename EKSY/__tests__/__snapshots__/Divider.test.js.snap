// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divider renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#e1e8ee",
        "height": 1,
      },
      Array [
        Object {
          "backgroundColor": "rgb(112,112,112)",
          "height": 1,
          "margin": 3,
          "marginLeft": 10,
          "marginRight": 10,
        },
        undefined,
      ],
    ]
  }
/>
`;
